lib/mparticle_flutter_sdk.dart

```diff
@@ -62,6 +65,45 @@ class MparticleFlutterSdk {
     return await _channel.invokeMethod('isKitActive', {'kitId': kit});
   }
 
+  /// Logs a product commerce event with an [productActionType], a promotion commerce event with a [eventType], and an impression commerce event if neither of the prior are implemented.
+  Future<void> logCommerceEvent({
```

Comment:
As I was implementing the web side of things, I realized that we should also have `customAttributes`, and `customFlags` in this function signature.